src/transformers/models/speech_to_text/modeling_speech_to_text.py

```diff
@@ -377,14 +378,14 @@ def forward(
         attention_mask: torch.Tensor,
         layer_head_mask: torch.Tensor,
         output_attentions: bool = False,
-    ):
+    ) -> torch.Tensor:
         """
         Args:
-            hidden_states (`torch.FloatTensor`): input to the layer of shape `(seq_len, batch, embed_dim)`
+            hidden_states (`torch.FloatTensor`): input to the layer of shape *(seq_len, batch, embed_dim)*
             attention_mask (`torch.FloatTensor`): attention mask of size
-                `(batch, 1, tgt_len, src_len)` where padding elements are indicated by very large negative values.
+                *(batch, 1, tgt_len, src_len)* where padding elements are indicated by very large negative values.
             layer_head_mask (`torch.FloatTensor`): mask for attention heads in a given layer of size
-                `(config.encoder_attention_heads,)`.
+                *(encoder_attention_heads,)*.
```

Comment:
Might be worth fixing the docstring in mBART as those should be code-blocks, not italics.

Reply:
Thanks for flagging! Resolved on main in https://github.com/huggingface/transformers/pull/20835 and in this PR in 01819c1.